demo/external-logging-elasticsearch/README.md

```diff
@@ -8,34 +8,62 @@ all its upstream dependencies.
 It also bundles auto-configuration System Groovy scripts, so that the WAR file starts
 up with pre-configured Logstash plugin settings and some other configs.
 
+Features of the demo:
+
+* Pipeline jobs logging goes to Elasticsearch
+* When tasks are executed on agents, the logs get posted to Elasticsearch directly
+  without passing though the master and causing scalability issues
+* Pipeline jobs override standard Log actions in the Jenkins core, so the
+  underlying implementation is transparent to users
+* Secrets are escaped in stored/displayed logs when running on master and agents.
```

Comment:
As in [the original](https://github.com/jglick/logstash-plugin/blob/logs-JENKINS-38381/demo/README.md) you could also mention

* console annotations work
* blocks are collapsible in **Console** screen
* origin container ID of every message is visible in Kibana (if you have set that up?) via `sender` field